pkg/registries/adapters/adapter.go

```diff
@@ -130,3 +142,45 @@ func imageToSpec(log *logging.Logger, req *http.Request, apbtag string) (*apb.Sp
 
 	return spec, nil
 }
+
+func isCompatibleVersion(specVersion string, minVersion string, maxVersion string) bool {
+	specMajorVersion, err := strconv.Atoi(strings.Split(specVersion, ".")[0])
+	if err != nil {
+		return false
+	}
+	specMinorVersion, err := strconv.Atoi(strings.Split(specVersion, ".")[1])
+	if err != nil {
+		return false
+	}
+	minMajorVersion, err := strconv.Atoi(strings.Split(minVersion, ".")[0])
+	if err != nil {
+		return false
+	}
+	minMinorVersion, err := strconv.Atoi(strings.Split(minVersion, ".")[1])
+	if err != nil {
+		return false
+	}
+	maxMajorVersion, err := strconv.Atoi(strings.Split(maxVersion, ".")[0])
+	if err != nil {
+		return false
+	}
+	maxMinorVersion, err := strconv.Atoi(strings.Split(maxVersion, ".")[1])
+	if err != nil {
+		return false
+	}
+
+	if minMajorVersion == maxMajorVersion {
+		if specMajorVersion >= minMajorVersion && specMajorVersion <= maxMajorVersion {
+			if specMinorVersion >= minMinorVersion && specMinorVersion <= maxMinorVersion {
+				return true
+			}
+		}
+	} else if minMajorVersion < maxMajorVersion {
```

Comment:
I think this is going to return false for `isCompatibleVersion("2.5", "1.0", "3.6")`
I don't know if this use case make sense or not.

Reply:
+1 good catch.